### PR TITLE
Cleanup Scalar Converter Code

### LINKED ISSUE #137: Cleanup Scalar Converter Code
We have the original draft of the scaler code uploaded in [this branch](https://github.com/microsoft/hummingbird/tree/cleanup/scaler). All credit here goes to @scnakandala, the original author and brains behind this.

It contains an un-edited implementation of RobustScaler, MaxAbsScaler, MinMaxScaler, and StandardScaler [here](https://github.com/microsoft/hummingbird/blob/cleanup/scaler/hummingbird/ml/operator_converters/scaler.py).  We need to integrate this old code into the current code.

There is a single test file [here](https://github.com/microsoft/hummingbird/blob/cleanup/scaler/tests/test_sklearn_scaler_converter.py) that needs to be cleaned up, expanded, and passing.

### PR DESCRIPTION
**Pull request by ahmedkrmn**
Closes #137 

- Update `__init__.py` and `supported.py`
- Modify the tests to use randomly generated arrays
- Add more tests

### CODE REVIEW

File: doc/html/hummingbird/ml/supported.html [hunk 1]
```diff
@@ -81,6 +81,10 @@ <h1 class="title">Module <code>hummingbird.ml.supported</code></h1>
 LogisticRegression,
 LogisticRegressionCV,
 Normalizer,
+RobustScaler,
```
Comment (interesaaat) [inline doc/html/hummingbird/ml/supported.html]: We are trying to get all ops alphabetically ordered. Can you please properly reorder the new operators?

File: doc/html/hummingbird/ml/supported.html [hunk 2]
```diff
@@ -156,6 +166,10 @@ <h1 class="title">Module <code>hummingbird.ml.supported</code></h1>
             SVC,
             # Preprocessing
             Normalizer,
+            RobustScaler,
```
Comment (interesaaat) [inline doc/html/hummingbird/ml/supported.html]: Alphabetically ordered here please.

File: doc/html/hummingbird/ml/supported.html [hunk 3]
```diff
@@ -131,7 +135,13 @@ <h1 class="title">Module <code>hummingbird.ml.supported</code></h1>
         from sklearn.svm import LinearSVC, SVC, NuSVC
 
         # Preprocessing
-        from sklearn.preprocessing import Normalizer
+        from sklearn.preprocessing import (
+            Normalizer,
```
Comment (interesaaat) [inline doc/html/hummingbird/ml/supported.html]: Alphabetic ordering here please.

File: hummingbird/ml/operator_converters/skl_scaler.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+            self.offset = torch.nn.Parameter(torch.FloatTensor([offset]), requires_grad=False)
+        else:
+            self.offset = None
+
+        if scale is not None:
+            self.scale = torch.nn.Parameter(torch.FloatTensor([scale]), requires_grad=False)
+        else:
+            self.scale = None
+
+        self.regression = False
```
Comment (interesaaat) [inline hummingbird/ml/operator_converters/skl_scaler.py]: No need to set this, regression is False by default.

File: hummingbird/ml/operator_converters/skl_scaler.py [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import torch
+import numpy as np
+from onnxconverter_common.registration import register_converter
+
+
+class Scaler(torch.nn.Module):
```
Comment (interesaaat) [inline hummingbird/ml/operator_converters/skl_scaler.py]: All operators now should extend [_base_operator](https://github.com/microsoft/hummingbird/blob/master/hummingbird/ml/operator_converters/_base_operator.py) to get the proper regression, etc properties. You can look at [normalizer](https://github.com/microsoft/hummingbird/blob/71119de8f4dc36b2158d3ef8c5859994ec9860de/hummingbird/ml/operator_converters/skl_normalizer.py#L12) as an example.

File: hummingbird/ml/supported.py [hunk 6]
```diff
@@ -25,6 +25,10 @@
 LogisticRegression,
 LogisticRegressionCV,
 Normalizer,
+RobustScaler,
```
Comment (interesaaat) [inline hummingbird/ml/supported.py]: Alphabetic order here.

File: hummingbird/ml/supported.py [hunk 7]
```diff
@@ -100,6 +104,10 @@ def _build_sklearn_operator_list():
             SVC,
             # Preprocessing
             Normalizer,
+            RobustScaler,
```
Comment (interesaaat) [inline hummingbird/ml/supported.py]: Alphabetic order.

**Comment by interesaaat** [comment]
Thanks @ahmedkrmn!
